### ISSUE TITLE
[Things] ...default values

### ISSUE THREAD

**Issue by im-abdullah** [raycast/extensions#15352]
### Extension

https://www.raycast.com/loris/things

### Description

I would like have default project and date always and cut the friction of the need to insert them over and over again every time I need to insert a new todo. 

### Who will benefit from this feature?

Every power user

**Comment by vimtor** [comment]
Things 3 has a setting to set the default destination to Today or Inbox. That could be a simpler solution.

**Comment by im-abdullah** [comment]
Yes I have it set as today in things but that doesn’t  reflect back on raycast extension.

**Comment by vimtor** [comment]
@raycastbot reopen this issue
